### ISSUE TITLE
Build: npm start doesn't work on its own

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#25848]
#### Steps to reproduce

1. clone repository
2. optionally run `npm run distclean` for existing install
3. run `npm start`

note - workaround is to run `npm install` first

#### What I expected

calypso

#### What happened instead

```
$ npm start

> wp-calypso@0.17.0 prestart /Users/alisterscott/Projects/a8c/wp-calypso
> run-s prestart:check-version install-if-deps-outdated prestart:welcome

sh: run-s: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! wp-calypso@0.17.0 prestart: `run-s prestart:check-version install-if-deps-outdated prestart:welcome`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the wp-calypso@0.17.0 prestart script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alisterscott/.npm/_logs/2018-07-03T04_47_00_081Z-debug.log

```

#### Browser / OS version

n/a

#### Screenshot / Video

n/a

#### Context / Source

Local development

**Comment by alisterscott** [comment]
cc @Automattic/team-calypso

**Comment by alisterscott** [comment]
`npm start` generates changes to the shrinkwrap file on master

**Comment by jorgecasariego** [comment]
@alisterscott I had the same problem and I will show you the steps I took to reach the solution:

1. Install nvm
curl -o-- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash

2. nvm install node

3. nvm use --delete-prefix v10.5.0

4. npm start 

I can't go into detail as I'm not an expert on Node but those were the steps I did to then run calypso.localhost:3000 in my browser.

I hope this can help

**Comment by jsnajdr** [comment]
This is a regression introduced in #25764 (by @blowery). The `prestart` script runs `install-if-deps-outdated` using `run-s`, but `run-s` (part of `npm-run-all`) is not installed yet...

Before that patch, `run-s` wasn't used before the initial install.

**Comment by alisterscott** [comment]
Fix in #25934 works well for me 👍
